<?xml version="1.0" encoding="UTF-8" ?>
<testsuites>
 <testsuite name="Tictactoe API" timestamp="2017-12-13T16:12:23" hostname="localhost" time="5.001" errors="0" tests="1" skipped="0" disabled="0" failures="1">
  <testcase classname="Tictactoe API" name="should be able to play one game to end" time="5.001">
   <failure type="exception" message="Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL."><![CDATA[Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
    at Timeout._onTimeout (/home/alli/week3/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4281:23)
    at ontimeout (timers.js:475:11)
    at tryOnTimeout (timers.js:310:5)
    at Timer.listOnTimeout (timers.js:270:5)]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>